examples/applications/Perera_Science_Granda_Nature_Suzuki/Granda_Perera_ml_example.ipynb

```diff
@@ -0,0 +1,2609 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example application of ORD datasets for downstream machine learning modeling"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Reproducing neural network predictor of reaction yields from Granda et al. Nature 2018, 559, 377, DOI: 10.1038/s41586-018-0307-8.<br>\n",
+    "Data from Perera et al. Science 2018, 359, 429, DOI: 10.1126/science.aap9112.<br>\n",
+    "ORD dataset location: https://github.com/open-reaction-database/ord-data/blob/main/data/68/ord_dataset-68cb8b4b2b384e3d85b5b1efae58b203.pb.gz<br>\n",
+    "ORD dataset ID: ord_dataset-68cb8b4b2b384e3d85b5b1efae58b203"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import modules\n",
+    "import ord_schema\n",
+    "from ord_schema import message_helpers, validations\n",
+    "from ord_schema.proto import dataset_pb2\n",
+    "\n",
+    "import math\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import tensorflow as tf\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os\n",
+    "import wget\n",
+    "\n",
+    "from rdkit import Chem\n",
+    "from rdkit.Chem import AllChem\n",
+    "from sklearn import model_selection, metrics\n",
+    "from glob import glob"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download dataset from ord-data\n",
+    "url = 'https://github.com/open-reaction-database/ord-data/blob/main/data/68/ord_dataset-68cb8b4b2b384e3d85b5b1efae58b203.pb.gz?raw=true'\n",
```

Comment:
How do you construct this URL for an arbitrary dataset? Is there a web UI where we could get that link?